data/huion-new-1060-plus.tablet

```diff
@@ -0,0 +1,23 @@
+# HUION
+# New 1060 Plus
+#
+
+[Device]
+Name=Huion New 1060 Plus
+ModelName=
```

Comment:
Is it okay to leave this empty? What should go here?

Reply:
if the device has a technical model name it should go there. All wacom devices have marketing name "Intuos Pro blah blah" and a model name in the specs "PTK-1234". If the HUION doesn't have one, leave it out.